server/src/main/java/umm3601/Server.java

```diff
@@ -16,7 +16,6 @@
 
   static String appName = "UMM CSci 3601 Lab 4";
```

Comment:
We presumably want to change this, or at least add a comment encouraging them to do this. It would be nice to have a list of all the places where they should add/switch to the name of _their_ project.

Reply:
I just made an issue (#3) for adding the list of places where the project name appears.